extensions/builtbybit/src/get-notifications.tsx

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from "react";
+import { Alert, AlertType, ContentType, ContentTypeNames, ContentTypeURLMap } from "./types/alert";
+import { AlertsService } from "./services/alertsService";
+import { showFailureToast, useCachedPromise } from "@raycast/utils";
+import { UserUtils } from "./utils/userUtils";
+import { Clipboard, Color, Icon, MenuBarExtra, open, showToast, Toast } from "@raycast/api";
+import { formatRelativeDate } from "./utils/dateUtils";
+
+export default function Command() {
+  const [enrichedAlerts, setEnrichedAlerts] = useState<Alert[]>([]);
+
+  const {
+    data: alertResponse,
+    isLoading,
+    revalidate,
+  } = useCachedPromise(() => AlertsService.fetchAlerts(), [], {
+    keepPreviousData: false,
```

Comment:
You might be able to use parseResponse or mapResults instead of below useEffetch